source/val/validation_state.h

```diff
@@ -268,9 +268,17 @@ class ValidationState_t {
     return id_decorations_[id];
   }
   const std::vector<Decoration>& id_decorations(uint32_t id) const {
+    // TODO: This would throw or generate SIGABRT if id has no
```

Comment:
We don't know the assumed contract for this function since there is no method comment.  :-)
Not your original commit so I'm just pleased you added this TODO. Thanks.